### DISCUSSION TITLE
mp3 files rendered as text

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44383] [Mobile]
GitHub Mobile beta
https://github.com/neonbjb/tortoise-tts/blob/main/examples/favorite_riding_hood.mp3
![Screenshot_20230115-071014~2.png](https://user-images.githubusercontent.com/2119212/212539800-88c03072-dd0f-4709-a6b9-b89babdeb2a8.png)

Most file types appear to show a "this is binary" message just like in the browser. But mp3 doesn't.

**Comment by rashidwassan** [comment]


**Reply by jsoref** [reply]
This is what I see in a web browser:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/2119212/212557807-d3f1c437-60b8-4f7e-b94f-28390106db27.png">

**Reply by jsoref** [reply]
This is what GitHub Mobile beta shows for an ipynb file:
![Screenshot_20230115-124659.png](https://user-images.githubusercontent.com/2119212/212557843-253ed879-022f-4220-943c-2f6f7b9e699c.png)

And it's what I'd expect to see for an mp3 file.

**Reply by candaceblewett42** [reply]
![image](https://user-images.githubusercontent.com/82898064/212567878-26a4166c-74f7-45dc-b5ff-ae6b0d53d21d.png)
